onnx/onnx.proto

```diff
@@ -67,25 +57,49 @@ message AttributeProto {
   optional bytes s = 4;               // UTF-8 string
   optional TensorProto t = 5;         // tensor value
   optional GraphProto g = 6;          // graph
-
+  optional ValueProto v = 12;         // value - subsumes everything but graph
+  
   repeated float floats = 7;          // list of floats 
   repeated int64 ints = 8;            // list of ints
   repeated bytes strings = 9;         // list of UTF-8 strings
   repeated TensorProto tensors = 10;  // list of tensors
   repeated GraphProto graphs = 11;    // list of graph
 }
 
+// To be compatible with both proto2 and proto3, we will use a version number
+// that is not defined by the default value but an explicit enum number.
+enum Version {
```

Comment:
Where is this enum used? I couldn't find a reference to it.

Reply:
It gets emitted by the protoc compiler and is available as a constant in language bindings.  Look fro IR_VERSION in the .py files.